action.yml

```diff
@@ -0,0 +1,135 @@
+name: 'deploy-infrastructure'
+author: '1up-team'
+description: 'Action to plan and deploy Terraform modules'
```

Comment:
Action to plan and deploy Terraform modules -> Action to plan and deploy Terraform stacks

Actually, module and stack are slightly different components of terraform